### PR TITLE
WIP:cri: Enable ipv6 explicitly

### LINKED ISSUE #2002: Try to enable ipv6 in CI to see if container 1.3.0 CI passes.
The script to install sample CNI config in the CRI repo now
expplicitly installs ipv6 routes for containerd 1.3.0.
So enable sysctl settingg for ipv6.

### PR DESCRIPTION
**Pull request by amshinde**
The script to install sample CNI config in the CRI repo now
expplicitly installs ipv6 routes for containerd 1.3.0.
So enable sysctl settingg for ipv6.

Fixes #2002

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>

### CODE REVIEW

**Comment by amshinde** [comment]
Test to check if https://github.com/kata-containers/runtime/pull/2100 passes with this PR.

File: integration/containerd/cri/integration-tests.sh [hunk 1]
```diff
@@ -72,6 +72,8 @@ ci_config() {
 		(
 		echo "Install cni config for cri-containerd test"
 		cd "${GOPATH}/src/${cri_containerd_repo}"
+		# Below script installing CNI config for containerd 1.3.0 installs ipv6 routes, so enable ipv6
+		sudo sysctl net.ipv6.conf.all.disable_ipv6=0
```
Comment (devimc) [inline integration/containerd/cri/integration-tests.sh]: I wonder if this should be part of a file, i,e https://github.com/kata-containers/tests/blob/master/.ci/install_kubernetes.sh#L81

Comment (jodh-intel) [reply integration/containerd/cri/integration-tests.sh]: If this is purely for k8s, I agree. Could you also add an `info "enabling IPv6"` message too?

File: integration/kubernetes/init.sh [hunk 2]
```diff
@@ -77,7 +77,9 @@ if ip a show "$cni_interface"; then
 fi
 
 echo "Start ${cri_runtime} service"
-sudo systemctl start ${cri_runtime}
+sudo systemctl daemon-reload
+sudo systemctl cat "${cri_runtime}.service"
```
Comment (devimc) [inline integration/kubernetes/init.sh]: debugging ?

File: integration/containerd/cri/integration-tests.sh (hunk 1 above)
Comment (amshinde) [reply integration/containerd/cri/integration-tests.sh]: Sure.

File: integration/kubernetes/init.sh (hunk 2 above)
Comment (amshinde) [reply integration/kubernetes/init.sh]: Yes, I am still figuring out some things while looking at the CI results. Will mark this as WIP till then.

**Pull request closed without merging**